try {} catch(e) {}

try {} catch(_e) {}

try {} catch(_) {}

try {} catch(err) {}

try {} catch(specificError) {}

try {} catch(_specificError) {}
